.changeset/@graphql-codegen_client-preset-9137-dependencies.md

```diff
@@ -0,0 +1,5 @@
+---
+"@graphql-codegen/client-preset": patch
+---
+dependencies updates:
+  - Updated dependency [`@graphql-typed-document-node/core@3.2.0-alpha-20230309104331-975d9d2` ↗︎](https://www.npmjs.com/package/@graphql-typed-document-node/core/v/3.2.0) (from `3.1.2`, in `dependencies`)
```

Comment:
TODO: update with the latest version when a PR  to `@graphql-typed-document-node/core` is accepted and released